lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -78,6 +78,19 @@ def date_range(self, start: AnyStr, end: AnyStr) -> list:
             date_range.append(f"{m.year:04}-{m.month:02}")
         return date_range
 
+    @pytest.mark.parametrize(
+        "malformed_token", ("malformed", "Bearer token"),
```

Comment:
OK, that's interesting; yes, "Bearer token" should fail because it's not a decodable JWT token, and that's great, but it actually wasn't what I meant. I had deliberately misspelled "bear(er)" in order to see what happened in our authorization code. 😀